mw-webapp/src/component/modal/ModalContent/ModalContent.module.scss

```diff
@@ -24,7 +24,7 @@ $modal-animation-time: 150ms;
   overflow: auto;
```

Comment:
line 5 - please convert to hex and create token (we use overlayColor for all modals? sidebar and etc/ Don't forget to add in the figma please

Reply:
done